db/migrate/20150227103048_add_project_id_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddProjectIdToLikes < ActiveRecord::Migration
+  def change
+    add_column :likes, :project_id, :integer
```

Comment:
You could have done this here `db/migrate/20150227101958_create_likes.rb` initially